R/plotSeries.R

```diff
@@ -0,0 +1,147 @@
+#' Plot Series
+#'
+#' Function that plots core taxa against time. #####################################
```

Comment:
Th has to be a complete sentence. Can you formulated it in a way that explains the general applicability for any series information?